site3/website/src/pages/release-notes.md

```diff
@@ -1,5 +1,69 @@
+* Close journal channel in testJunkEndedJournal https://github.com/apache/bookkeeper/pull/3307
+* Fix jvm_memory_direct_bytes_used metrics when using jdk11+ https://github.com/apache/bookkeeper/pull/3252
+* Autorecovery does not process underreplicated empty ledgers https://github.com/apache/bookkeeper/pull/3239
+* Fix wrong ledger id parse radix for index relocation file in IndexPersistenceMgr https://github.com/apache/bookkeeper/pull/2944
+* Fix the infinite waiting for shutdown due to throttler limit https://github.com/apache/bookkeeper/pull/2942
+* Make sure the LedgerHandle close callback can be completed when encounter exception https://github.com/apache/bookkeeper/pull/2913
+
+#### Improvements
+
+* If ensembleList is empty, return PlacementPolicyAdherence.FAIL. https://github.com/apache/bookkeeper/pull/3369
+* reduce unnecessary checkpoints https://github.com/apache/bookkeeper/pull/3341
+* Pre break loop when self create layoutZNode succeed. https://github.com/apache/bookkeeper/pull/3335
+* Avoid init WriteSet when waitForWriteSetMs < 0. https://github.com/apache/bookkeeper/pull/3325
+* Tuing PendingReadOp.java seq https://github.com/apache/bookkeeper/pull/3330
+* Apply the backpressure changes on the V2 requests https://github.com/apache/bookkeeper/pull/3324
+* Deduplicate error log for SSLException https://github.com/apache/bookkeeper/pull/3320
+* Switch to rely on SslEngine for Hostname Verification https://github.com/apache/bookkeeper/pull/3310
+* Try to use Java9 CRC32C when JNI based CRC is not available https://github.com/apache/bookkeeper/pull/3309
+* Enhance future sync wait. https://github.com/apache/bookkeeper/pull/3336
+* validate diskUsageThreshold and diskUsageLwmThreshold https://github.com/apache/bookkeeper/pull/3285
+* Replace unsafe NoEntryException with IOException https://github.com/apache/bookkeeper/pull/2909
+* let bookie quit if journal thread exit https://github.com/apache/bookkeeper/pull/2887
+* Ledger replicate supports throttle https://github.com/apache/bookkeeper/pull/2778
+* minorCompactionInterval should be greater than gcWaitTime https://github.com/apache/bookkeeper/pull/2116
+* add metric cbThreadPoolQueueSize https://github.com/apache/bookkeeper/pull/3424
+* Optimize concurrent collection's shrink logic https://github.com/apache/bookkeeper/pull/3417
+* Make BookieFileChannel interface public https://github.com/apache/bookkeeper/pull/3396
+* Prioritize compaction of entry logs with the lowest amount of remaining usable data https://github.com/apache/bookkeeper/pull/3390
+* Switch back ordered executor to LinkedBlockingQueue https://github.com/apache/bookkeeper/pull/3384
+* Consolidate Netty channel flushes to mitigate syscall overhead https://github.com/apache/bookkeeper/pull/3383
+* Shut down ReplicationWorker and Auditor on non-recoverable ZK error https://github.com/apache/bookkeeper/pull/3374
+* add journalCbQueueLatency https://github.com/apache/bookkeeper/pull/3364
```

Comment:
I suggest moving this and other metrics-related PRs under separate section "Metrics changes" (same level as "Improvements") for better visibility